dark/index.html

```diff
@@ -150,7 +150,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
-        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
+        <a href="https://img.icons8.com/fluency/48/telegram-app.png" target="_blank "><img
             src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>

```

Comment:
Change this to following @maddy020 

```suggestion
            src="https://img.icons8.com/fluency/48/telegram-app.png" alt="Telegram Icon" /></a>
```